apps/test/unit/code-studio/components/progress/TeacherPanelProgressBubbleTest.jsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {assert, expect} from '../../../../util/reconfiguredChai';
+import {TeacherPanelProgressBubble} from '@cdo/apps/code-studio/components/progress/TeacherPanelProgressBubble';
+import color from '@cdo/apps/util/color';
+import {LevelKind, LevelStatus} from '@cdo/apps/util/sharedConstants';
+
+const defaultProps = {
+  level: {
+    assessment: null,
+    contained: false,
+    driver: null,
+    isConceptLevel: false,
+    levelNumber: 4,
+    navigator: null,
+    paired: null,
+    passed: false,
+    status: LevelStatus.not_tried,
+    user_id: 1
+  },
+  inMiniRubricExperiment: false
+};
+
+describe('StudentTable', () => {
+  it('shows pair programming logo if level is paired as navigator', () => {
+    const wrapper = shallow(
+      <TeacherPanelProgressBubble
+        {...defaultProps}
+        level={{
+          ...defaultProps.level,
+          paired: true,
+          driver: 'My Friend'
+        }}
+      />
+    );
+
+    expect(wrapper.find('FontAwesome')).to.have.length(1);
+  });
+  it('shows pair programming logo if level is paired as driver', () => {
+    const wrapper = shallow(
+      <TeacherPanelProgressBubble
+        {...defaultProps}
+        level={{
+          ...defaultProps.level,
+          paired: true,
+          navigator: 'My Friend'
+        }}
+      />
+    );
+
+    expect(wrapper.find('FontAwesome')).to.have.length(1);
+  });
+  it('shows number in bubble if not paired', () => {
+    const wrapper = shallow(<TeacherPanelProgressBubble {...defaultProps} />);
+
+    expect(wrapper.find('FontAwesome')).to.have.length(0);
```

Comment:
The name of this test made me think that you would be testing whether the number _is_ there and is the correct number, not just that the FontAwesome icon is _not_ there.